Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build the manager binary
-FROM golang:1.17 as builder
+FROM golang:1.19 as builder
```

Comment:
You can use GOARCH with 1.17. What specifically about 1.19 is different for supporting different architecture?
![image](https://user-images.githubusercontent.com/11228024/182708990-6ebe7f00-477e-4c87-a3e2-c2a608eaa845.png)

<img width="1278" alt="image" src="https://user-images.githubusercontent.com/11228024/182709057-355bb1f2-2e5e-4426-96cf-0fe6ba3f4b46.png">

src https://hub.docker.com/_/golang?tab=tags&page=1&name=1.17&ordering=-name
```suggestion
FROM golang:1.17 as builder
```

Reply:
looks like @shubham-pampattiwar is making similar change at https://github.com/konveyor/volume-snapshot-mover/pull/131

Reply:
yes. I'll close this PR. Thank you!